clients/admin-ui/src/features/privacy-notices/cells.tsx

```diff
@@ -3,14 +3,30 @@ import React from "react";
 import { CellProps } from "react-table";
 
 import { PRIVACY_NOTICE_REGION_MAP } from "~/features/common/privacy-notice-regions";
-import { EnableCell, MapCell, MultiTagCell } from "~/features/common/table/";
+import { EnableCell } from "~/features/common/table/";
 import { MECHANISM_MAP } from "~/features/privacy-notices/constants";
 import { usePatchPrivacyNoticesMutation } from "~/features/privacy-notices/privacy-notices.slice";
 import { PrivacyNoticeResponse } from "~/types/api";
 
 export const MechanismCell = (
-  cellProps: CellProps<typeof MECHANISM_MAP, string>
-) => <MapCell map={MECHANISM_MAP} {...cellProps} />;
+  cellProps: CellProps<PrivacyNoticeResponse, string>
+) => {
+  const { row } = cellProps;
+  const tagValue = MECHANISM_MAP.get(row.original.consent_mechanism);
+  return (
+    <Badge
```

Comment:
I think it might make sense to change `MapCell` to use this badge style instead of the tag it originally used. that seems to be the way the new designs are leaning. then you wouldn't need to make this change to `MechanismCell` and it can still be a wrapper around `MapCell`, and we wouldn't need to repeat this `<Badge />` code for locations either

`MapCell` is used in one other place, which is in the custom fields table but I feel like that could probably use this new design too?
![image](https://github.com/ethyca/fides/assets/24641006/06fafe34-dd3c-45e9-bca2-2646b5bf593a)

if custom fields does need to stay the same, then we can still go with this solution, it's just not as dry as it could be :)

Reply:
You're right I reverted MechanismCell back to use MapCell and updated MapCell